cloudsync/tests/test_oauth_redir_server.py

```diff
@@ -54,9 +71,31 @@ def send_req():
 
     shutdown_signal.wait()
     srv.shutdown()
+    srv.server_close()
+    assert server_close_calls == 1
```

Comment:
To confirm, the `EventApiServer.server_close` method gets called synchronously, and not in another thread, so this won't be timing sensitive?

Reply:
correct. actually they are both synchronous in this test. `shutdown()` can be called from a server thread - that is covered in test_oauth.py